src/Apps/Authentication/Utils/helpers.ts

```diff
@@ -177,17 +176,11 @@ export const maybeUpdateRedirectTo = (
 }
 
 export const setCookies = options => {
-  const { afterSignUpAction, destination } = options
+  const { afterSignUpAction } = options
 
   if (afterSignUpAction) {
     Cookies.set(AFTER_AUTH_ACTION_KEY, JSON.stringify(afterSignUpAction))
   }
-
-  if (destination) {
-    Cookies.set("destination", destination, {
-      expires: 60 * 60 * 24,
-    })
-  }
```

Comment:
This was the only real thing this option did. Which is never actually checked anywhere.